api/api.py

```diff
@@ -2,15 +2,62 @@
 Whether it be an amateur developer or a seasoned researcher, this service should be easy to make the most out of through its endpoints."""
 from dotenv import load_dotenv
 from flask import Flask, request
+import psycopg2
+import psycopg2.extras
+from psycopg2.extensions import connection, cursor
+from os import environ as env
 
 app = Flask(__name__)
```

Comment:
unsure on this but, can this be included inside `if __name__ == "__main__":`?

Reply:
To my knowledge, the app needs to be a global object so that it can be referenced in the '@app.route' wrapper. This helps separate helper/external functions from the endpoints of our api

Reply:
sounds good to me